Dockerfile

```diff
@@ -144,7 +150,7 @@ RUN git init . && git remote add origin "https://github.com/go-delve/delve.git"
 ARG DELVE_VERSION=v1.21.1
 RUN git fetch -q --depth 1 origin "${DELVE_VERSION}" +refs/tags/*:refs/tags/* && git checkout -q FETCH_HEAD
 
-FROM base AS delve-build
+FROM base AS delve-supported
```

Comment:
A better approach would be if every dependency defined

```
FROM.. AS dep-build
RUN compile -o /build/ 

FROM scratch AS dep
COPY --from=dep-build /build /

..
COPY --from=dep --link / /usr/bin/
```

Then every target can be built and tested separately and there is no need for these "dummy" stages. But I didn't change it as previous pattern is used for all deps.